lib/features/home_screen/home_page.dart

```diff
@@ -7,8 +17,81 @@ class HomePage extends StatefulWidget{
 
 class _HomePageState extends State<HomePage>{
 
+  final HomeCubit homeCubit = Modular.get<HomeCubit>();
+
+  @override
+  void initState(){
+    super.initState();
+    homeCubit.fetchComics();
+  }
+
  @override
  Widget build(BuildContext context){
-   return const Center(child: Text("Home Page"),);
+   return Scaffold(
+     appBar: AppBar(
+       backgroundColor: Colors.white,
+       centerTitle: false,
+       title: const Text(
+         HomePageStrings.appBarTitle,
+         style: TextStyle(
+           fontWeight: FontWeight.bold,
+           fontSize: HomePageDimens.appBarTextSize,
+           color: Colors.black,
+         ),
+       ),
+     ),
+     body: Padding(
+       padding: const EdgeInsets.all(HomePageDimens.aroundBodyPadding),
+       child: _buildComicsList(),
+     ),
+   );
  }
-}
\ No newline at end of file
+
+ Widget _buildComicsList(){
+    return BlocProvider(
+      create: (_) => homeCubit,
+      child: BlocListener<HomeCubit, HomeState>(
+        listener: (context, state){
+          if (state is HomeError){
+            ScaffoldMessenger.of(context).showSnackBar(
+              SnackBar(content: Text(state.message ?? HomePageStrings.error)),
+            );
+          }
+        },
+        child: BlocBuilder<HomeCubit, HomeState>(
+          builder: (context, state){
+            if (state is HomeInitial) {
+            return LoadingWidget();
+            } else if (state is HomeLoading) {
+            return LoadingWidget();
+            } else if (state is HomeLoaded) {
```

Comment:
It's the same:
```            if (state is HomeInitial) {
            return LoadingWidget();
            } else if (state is HomeLoading) {
            return LoadingWidget();
            } else if (state is HomeLoaded) {
```

You can make something like that:
`if (state is HomeLoading || state is HomeInitial)`